src/concrete/ml/common/preprocessors.py

```diff
@@ -0,0 +1,999 @@
+"""Graph pre-processors for automatic rounding."""
```

Comment:
overall looks good but : 
- very few comments in the actual code, which makes the whole file hard to follow (cannot imagine debugging this in the future) 
- some remaining comments that are confusing / seem irrelevant